.github/workflows/release.yml

```diff
@@ -0,0 +1,49 @@
+name: Build Java-LibKiwix
+
+on:
+  push:
+    branches-ignore:
+      - '*'
+      - '*/**'
+    tags:
+      - '*'
+      - '*/**'
+
+jobs:
+  publish:
+
+    runs-on: ubuntu-18.04
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          distribution: adopt
+          java-version: 11
+
+      - name: Building Project
+        run: bash contrib/build.sh
+
+      - name: Release build
+        run: |
```

Comment:
For new folder structure we don't need to go into android-libkiwixbuild we should remove this. Now we can directly use ```./gradlew androidSourcesJar build```